src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -418,15 +421,47 @@ private async Task<JObject> GetLocalSettingsJsonAsJObjectAsync()
         public override async Task RunAsync()
         {
             await PreRunConditions();
+            var isVerbose = VerboseLogging.HasValue && VerboseLogging.Value;
```

Comment:
Can we move this logic that is specific to .NET to a different method and out of `RunAsync`? That will help declutter this method and make the code easier to navigate.

Reply:
This will also enable you to reduce the variables as you'll be able to drive the logic without needing to have an additional flag for OOP.